assets/js/modules/analytics/setup/setup-main.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Analytics Main setup component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { __, _x, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AccountCreate from './account-create-legacy';
+import SetupForm from './setup-form';
+import { SvgIcon } from '../../../util';
+import { STORE_NAME } from '../datastore';
+
+export default function SetupMain() {
+	const accounts = useSelect( ( select ) => select( STORE_NAME ).getAccounts() ) || [];
+	const isCreateAccount = useSelect( ( select ) => select( STORE_NAME ).isCreateAccount() );
```

Comment:
See other comment, this could just do the `ACCOUNT_CREATE` check against `getAccountID()`.